### PR TITLE
[FIX][8.0] Transfer Fiscal Position when creating a quotation

### LINKED ISSUE #44: Transfer Fiscal Position when creating a quotation
Correctly copy tax position while creating a quotation from a list of business requirements. Take the fiscal position of the customer set in the Business Requirement record.

### PR DESCRIPTION
**Pull request by astirpe**
Fixes #44

### CODE REVIEW

**Comment by elicoidal** [comment]
@astirpe thanks for your contribution.
:+1: 
@mmalorni

File: business_requirement_deliverable_crm/wizard/crm_make_sale.py [hunk 1]
```diff
@@ -53,6 +58,7 @@ def prepare_sale_order_line(self, case_id, order_id):
                     'product_uom': br_line.uom_id.id,
                     'product_uos': br_line.uom_id.id,
                     'price_unit': br_line.unit_price,
+                    'tax_id': [(6, 0, [x.id for x in taxes])],
```
Comment (pedrobaeza) [inline business_requirement_deliverable_crm/wizard/crm_make_sale.py]: Use directly `taxes.ids`

File: business_requirement_deliverable_crm/wizard/crm_make_sale.py [hunk 2]
```diff
@@ -44,6 +45,10 @@ def prepare_sale_order_line(self, case_id, order_id):
             if br.state in ('drop', 'cancel'):
                 continue
             for br_line in br.deliverable_lines:
+                taxes = br_line.product_id.taxes_id
+                fp = order.partner_id.property_account_position
+                if fp:
+                    taxes = fp.map_tax(taxes)
```
Comment (pedrobaeza) [inline business_requirement_deliverable_crm/wizard/crm_make_sale.py:51]: Filter per company: `fp.map_tax(taxes).filtered(lambda x: x.company_id == ...`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Anyway, why don't execute onchanges programatically to avoid all this manual stuff?

**Comment by elicoidal** [comment]
> Anyway, why don't execute onchanges programatically to avoid all this manual stuff?

Good point indeed

**Review by pedrobaeza — APPROVED** [review]

**Comment by astirpe** [comment]
Regarding the proposal of @pedrobaeza to make use of onchange methods, I would propose a new PR (as enhancement) to partially rewrite the code of this module

**Comment by pedrobaeza** [comment]
OK, I agree. It's out of the scope of this PR

**Comment by elicoidal** [comment]
OK with that too

**Comment by pedrobaeza** [comment]
Merging then

**Comment by elicoidal** [comment]
thanks @pedrobaeza @astirpe

**Pull request merged**